### ISSUE TITLE
Handle proxying requests properly

### ISSUE THREAD

**Issue by FireMasterK** [pluja/Parasitter#53]
![image](https://user-images.githubusercontent.com/20838718/92993586-64f0b580-f510-11ea-836d-ecbd90ff640d.png)

Ideally, you should urlencode it and up it into a parameter instead of using hacks such as replacing `/` with `~`, etc.

Also, what about using nginx for this?

**Comment by pluja** [comment]
Yeah, this is something that needs improvement. Adding to important list.
